charts/actions-runner-controller/templates/deployment.yaml

```diff
@@ -5,6 +5,8 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
     {{- include "actions-runner-controller.labels" . | nindent 4 }}
+  annotations:
+    seccomp.security.alpha.kubernetes.io/pod: runtime/default
```

Comment:
I know nothing about this, the controller and kube-proxy are based on a distroless images:
https://github.com/summerwind/actions-runner-controller/blob/master/Dockerfile
https://github.com/brancz/kube-rbac-proxy/blob/master/Dockerfile
does this actually do anything in the context of a distroless image?

Reply:
Yes, it provides sane defaults on which syscalls are allowed https://kubernetes.io/docs/tutorials/clusters/seccomp/#create-pod-that-uses-the-container-runtime-default-seccomp-profile

Reply:
Cool, needs testing. Is there a reason you have implemented it the < v.1.19 way instead of the => 1.19 way?

The doc says it should be implemented like this for V1.19 or later:

```yaml
spec:
  securityContext:
    seccompProfile:
      type: RuntimeDefault
```